docs/2_4_common_schema.adoc

```diff
@@ -449,17 +449,17 @@ _Note that `Hz` is typically used as `Unit.name` for a frequency quantity, but i
 
 _The `<BaseUnit>` definitions can be utilized for different purposes (the following application examples are optional and a tool may also completely ignore the `Unit` definitions):_
 
-Signal connection check::
+Variable connection check::
```

Comment:
"Variable connection" sounds a bit strange. 
What about: "Unit check when connecting variables of different FMUs"

Reply:
Agreed and changed.